packages/sit-onyx/src/components/OnyxNavItem/OnyxNavItem.vue

```diff
@@ -0,0 +1,124 @@
+<script lang="ts" setup>
+import { computed, ref } from "vue";
+import type { OnyxNavItemProps } from "./types";
+import OnyxListbox from "../OnyxListbox/OnyxListbox.vue";
+import type { ListboxOption } from "../OnyxListbox/types";
+
+const props = withDefaults(defineProps<OnyxNavItemProps>(), {
+  active: false,
+});
+
+const emit = defineEmits<{
+  /**
+   * Emitted when the navItem is clicked (via click or keyboard).
+   */
+  navigate: [href: string];
+}>();
+
+defineSlots<{
+  /**
+   * A default slot that can be used for OnyxBadge.
+   */
+  default(): unknown;
+}>();
+
+const listboxVisible = ref(false);
+
+const nestedOptions = computed(() => {
+  return props.options?.map((opt) => {
+    return { id: opt.href, label: opt.label };
+  }) satisfies ListboxOption[] | undefined;
+});
+
+function hideListbox() {
+  setTimeout(() => {
+    listboxVisible.value = false;
+  }, 500);
+}
+</script>
+
+<template>
+  <li
+    role="menuitem"
+    tabindex="0"
+    :aria-label="props.label"
+    :class="[
+      'onyx-nav-item',
+      'onyx-text',
+      { 'onyx-nav-item--active': props.active || props.options?.find((opt) => opt.active) },
+    ]"
+    @click="props.href && !props.options && emit('navigate', props.href)"
+    @keydown.enter="props.href && !props.options && emit('navigate', props.href)"
+    @mouseover="listboxVisible = true"
+    @focusin="() => {}"
+    @mouseleave="hideListbox()"
+    @blur="() => {}"
```

Comment:
```suggestion
    @mouseleave="hideListbox()"
```

They are not used, right?

Reply:
Yes but a11y requires them...